### PR TITLE
Add sub-second intervals

### PR DESCRIPTION
**Pull request by astro**

### CODE REVIEW

File: src/view.rs [hunk 1]
```diff
@@ -507,18 +507,18 @@ pub fn format_quantity(
...
         return format!("{}0{}s", number_style, unit_style);
     }
 
     let mut string = String::new();
 
-    for (unit, factor) in &[("h", 3600), ("m", 60), ("s", 1)] {
-        if seconds >= *factor {
-            string.push_str(&format!("{}{}{}{}", number_style, seconds / factor, unit_style, unit));
-            seconds %= factor;
+    for (unit, factor) in &[("h", 3600_000), ("m", 60_000), ("s", 1_000), ("ms", 1)] {
```
Comment (SuperSandro2000) [inline src/view.rs]:
```suggestion
    for (unit, factor) in &[("h", 3_600_000), ("m", 60_000), ("s", 1_000), ("ms", 1)] {
```
https://travis-ci.org/p-e-w/hegemon/jobs/653199292#L453

File: src/view.rs [hunk 2]
```diff
@@ -507,18 +507,18 @@ pub fn format_quantity(
 }
 
 fn format_duration(duration: Duration, number_style: impl Display, unit_style: impl Display) -> String {
-    let mut seconds = duration.as_secs();
+    let mut millis = duration.as_millis();
```
Comment (p-e-w) [inline src/view.rs]:
```suggestion
    let mut milliseconds = duration.as_millis();
```

I prefer fully writing out everything.

File: src/view.rs [hunk 3]
```diff
@@ -507,18 +507,18 @@ pub fn format_quantity(
...
     let mut string = String::new();
 
-    for (unit, factor) in &[("h", 3600), ("m", 60), ("s", 1)] {
-        if seconds >= *factor {
-            string.push_str(&format!("{}{}{}{}", number_style, seconds / factor, unit_style, unit));
-            seconds %= factor;
+    for (unit, factor) in &[("h", 3600_000), ("m", 60_000), ("s", 1_000), ("ms", 1)] {
+        if millis >= *factor {
+            string.push_str(&format!("{}{}{}{}", number_style, millis / factor, unit_style, unit));
+            millis %= factor;
```
Comment (p-e-w) [inline src/view.rs]: I don't think formatting sub-second durations as "100 ms" makes for a good UX. It's probably better to restrict intervals to multiples of 100 ms and then format as "0.2 s" and so on.

File: src/model.rs [hunk 4]
```diff
@@ -78,14 +78,17 @@ impl Application {
...
-                Interval::new(1, 10),
-                Interval::new(2, 10),
-                Interval::new(3, 10),
-                Interval::new(5, 12),
-                Interval::new(10, 12),
-                Interval::new(30, 10),
-                Interval::new(60, 10),
-                Interval::new(300, 12),
+                Interval::new(100, 10),
+                Interval::new(250, 20),
```
Comment (p-e-w) [inline src/model.rs]:
```suggestion
                Interval::new(200, 10),
```

**Comment by p-e-w** [comment]
Thank you for those high quality PRs. Much appreciated!

**Pull request merged**